requirements.txt

```diff
@@ -30,12 +30,12 @@ protobuf>=3.10.0,<3.10.99
 djangosaml2==0.17.2
 pysaml2==4.8.0
 
-SQLAlchemy>=1.3.10,<1.3.199
-psycopg2>=2.8.4,<2.8.499 --no-binary psycopg2
-mysqlclient>=1.4.4,<1.4.499
+SQLAlchemy>=1.3.10,<1.3.99
+psycopg2>=2.8.4,<2.8.99 --no-binary psycopg2
+mysqlclient>=1.4.4,<1.4.99
 google-cloud-bigquery>=1.21.0,<1.21.99
 
-pinax-eventlog>=2.0.3,<2.99
+pinax-eventlog>=2.0.3,<2.0.99
```

Comment:
The newest version of `pinax-eventlog` available is 3.0.0, https://pypi.org/project/pinax-eventlog/#history. Why haven't we upgraded to that?

Reply:
We can. Truthfully, I started working on updating the BE dependencies in October, and the major releases for `pinax-eventlog` and `whitenoise` had not come out yet (release dates in December). I suppose I should have done another check when I was finishing PR #842 in January. There are likely others that have had releases in Q4 of 2019 and Q1 of 2020. I'm not too afraid of updating these, but there is some consideration for waiting for a couple minor releases to come out so initial bugs are worked out. I consider this a separate issue. Would you like me to create an issue?

Reply:
Yes, I just did a scan, and there are about 14 libraries with newer versions (some major) that we could update.

Reply:
Ya... I know it's bit of more testing involved what change in the major release. Also, we need to have a full understanding of how the library used in the project for testing parts of it. Since not all of the code written by one person and it will be impractical to document the purpose of the library being used

Reply:
Still, it seems like a good practice to include an issue for each release cycle checking the BE dependencies and assessing whether we should upgrade or not.